### ISSUE TITLE
Win10: Reading CSD failed (No such file or directory)...stopping

### ISSUE THREAD

**Issue by chenliu52** [CsoundQt/CsoundQt#364]
Hi :)

I just switched to my PC from OS and installed Csound and CsoundQt, but I could not run any CSD files from my own or the examples. The console kept complaining Reading CSD failed (No such file or directory)...stopping. I followed the instruction and copy-pasted the files from the bin folder to C: -> Program Files -> Csound6_x64 -> bin.

I would really appreciate the help and support. Thank you!

**Comment by tarmoj** [comment]
hi!

But can you open a csd so that you see it in your CsoundQt editor?
What happens if you press "Run in Terminal"?
Does Csound work from command line (cmd.exe I think)?
What is the version of Csound and CsoundQt?

greetings,
tarmo

Kontakt chenliu52 ***@***.***>) kirjutas kuupäeval P, 14.
november 2021 kell 18:51:

**Comment by chenliu52** [comment]
Yes I can open a csd file and it shows up in the editor.
I just pressed run in terminal for the oscil.csd example and the command line popped up. It couldn't find the directory C:\Program Files\Csound6_x64\doc\manual\examples>
Csound 6.16.2 is and CsoundQt is 0.9.8.1

Thanks a lot for your support!

All the best,
Chen Liu.

**Comment by tarmoj** [comment]
Hi.

Good, this is the latest available Csound and CsoundQt.
This is very strange. Can it be that you use some unusual character
encoding for the system?  That you can open the file in the editor means
that the Qt side (the front-end) can deal with the file name but somehow
when it is passed to the core Csound, it sees it wrong.
I am not on Windows myself, I don't know it so well.
Can you try a couple of things:
- open terminal (cmd.exe) and type the command:
csound -odac "C:\Program Files\Csound6_x64\doc\manual\examples\oscil.csd"
(or some other file)
what if you copy some csd to another location, try some place where there
is no spaces in the file name?
Can you open a new file in CsoundQt (Ctrl+N) and try to run that?

Do you have chance to try it on another computer?

Hard to guess, I hope you will have some progress!

tarmo

Kontakt chenliu52 ***@***.***>) kirjutas kuupäeval P, 14.
november 2021 kell 19:54:

**Comment by chenliu52** [comment]
I guess that could be a problem because my PC was installed with a Chinese version of Windows 10. I also discovered that there are some unintelligent Chinese characters in the manual. 

However, I tried running the command in cmd.exe and it worked and run the score!

I tried to create a new csd called buzzTest on my external drive and had it open in Csound but it just couldn't run it. 

I also reinstalled everything to D drive but the same thing happened :((( Although it is still recommended to install them in C;\Program Files, isn't it?
